functions/src/scheduler/schedule.ts

```diff
@@ -23,3 +25,28 @@ export const deletePingsAfter30days: any = functions.pubsub.schedule('00 00 * *
     });
     return Promise.all(promises);
   });
+
+export const runAllMonitors60Mins: any = functions.pubsub.schedule('every 60 mins')
+  .onRun(async () => {
+    const snapshots: QuerySnapshot = await FirebaseAdmin.firestore()
+      .collection('projects')
+      .get();
+    
+    snapshots.docs.forEach(async (doc: firestore.QueryDocumentSnapshot) => {
+      const project: ProjectModel = <ProjectModel>doc.data();
+      const promises: Promise<WriteResult>[] = [];
+      try {
+        if (project.url && Array.isArray(project.monitors) && project.monitors.length > 0) {
+
+          for (const monitor of project.monitors) {
```

Comment:
I prefer `forEach` if possible to be consistent

Reply:
done